### PR TITLE
DFJR 971 - Collapses expandable on mobile

### PR DESCRIPTION
**Pull request by anselmbradford**
Expandables will collapse when page is loaded under 600px width, or if it is resized below 600px. If resized and collapsed, it won't remember that it was expanded at desktop sizes and won't expand when resizing back to desktop.

## Changes

- Makes Expandables collapse under 600px window size.
- General code comment cleanup.
- Removes `addEventListener` check since IE8 is no-js state now.

## Testing

- `gulp build`
- Create a browse filterable page in wagtail and add a filter.
- Check "Is Expanded" in wagtail admin.
- Load the page preview in a small window, note filter is collapsed.
- Load the page preview in a large window, note filter is expanded.
- Resize the page to a small size and note filter collapses.

## Review

- @kurtw 
- @sebworks 
- @jimmynotjim 
- @KimberlyMunoz 

## Todos

- @kurtw Can we add a note next to the "Is Expanded" checkbox in wagtail that says it won't apply under 600px wide?
- If filters have any user entered data, they need `is_expanded=true` passed to `block.value` in https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/browse-filterable/index.html#L34

### CODE REVIEW

**Comment by sebworks** [comment]
:+1:

File: cfgov/unprocessed/js/molecules/Expandable.js [hunk 1]
```diff
@@ -47,8 +50,10 @@ function Expandable( element ) { // eslint-disable-line max-statements, inline-c
    */
   function init( state ) {
     _calcHeight();
-    if ( state === EXPANDED ||
-         _dom.getAttribute( 'data-state' ) === 'expanded' ) {
+    // Even if expanded is set, don't expand if in mobile window size.
+    if ( !_isInMobile() &&
+         (state === EXPANDED ||
```
Comment (jimmynotjim) [inline cfgov/unprocessed/js/molecules/Expandable.js]: Minor, add space after `(`

**Comment by jimmynotjim** [comment]
LGTM

File: cfgov/unprocessed/js/molecules/Expandable.js (hunk 1 above)
Comment (anselmbradford) [inline cfgov/unprocessed/js/molecules/Expandable.js]: Fixed!

**Pull request merged**